tests/conftest.py

```diff
@@ -45,14 +51,25 @@ def pytest_ignore_collect(collection_path, path, config):
     These somtimes fail to import and cause pytest to fail.
     """
     if "tests/GalSim/tests" in str(collection_path):
-        if not any(
-            [t in str(collection_path) for t in test_config["enabled_tests"]["galsim"]]
-        ):
+        if (
+            not any(
+                [
+                    t in str(collection_path)
+                    for t in test_config["enabled_tests"]["galsim"]
+                ]
+            )
+        ) and "*" not in test_config["enabled_tests"]["galsim"]:
```

Comment:
It does never return True, meaning all tests get collected when "*" is used. I kept the infrastructure so that in the future if we need to skip things again, we can.